9.2/test/run

```diff
@@ -212,6 +212,71 @@ function run_tests() {
   test_postgresql $name
 }
 
+function run_change_password_test() {
+  local name="change_password"
+
+  local database='db'
+  local user='user'
+  local password='password'
+  local admin_password='adminPassword'
+
+  local volume_dir
+  volume_dir=`mktemp -d --tmpdir pg-testdata.XXXXX`
+  chmod a+rwx ${volume_dir}
+
+  DOCKER_ARGS="
+-e POSTGRESQL_DATABASE=${database}
+-e POSTGRESQL_USER=${user}
+-e POSTGRESQL_PASSWORD=${password}
+-e POSTGRESQL_ADMIN_PASSWORD=${admin_password}
+-v ${volume_dir}:/var/lib/pgsql/data
+" create_container ${name}
+
+  # need to set these because `postgresql_cmd` relies on global variables
+  USER=${user}
+  PASS=${password}
+
+  # need this to wait for the container to start up
+  CONTAINER_IP=$(get_container_ip ${name})
+  test_connection ${name}
+
+  echo "  Testing login"
+  assert_login_access ${user} ${password} true
+  USER='postgres'
+  PASS=${admin_password}
+  assert_login_access 'postgres' ${admin_password} true
+
+  echo "  Changing passwords"
+  docker stop $(get_cid ${name})
```

Comment:
For some reason my local tests fail after this, and I cannot start the new container getting:

> FATAL:  configuration file "/var/lib/pgsql/data/userdata/postgresql.conf" contains errors

Quickly glanced the file and it seemed ok, though.